cpp/thirdparty/versions.txt

```diff
@@ -23,8 +23,8 @@
 # `DEPENDENCIES` array (see the comment on top of the declaration for the
 # format).
 
-ARROW_ABSL_BUILD_VERSION=20211102.0
-ARROW_ABSL_BUILD_SHA256_CHECKSUM=dcf71b9cba8dc0ca9940c4b316a0c796be8fab42b070bb6b7cab62b48f0e66c4
+ARROW_ABSL_BUILD_VERSION=20230125.3
+ARROW_ABSL_BUILD_SHA256_CHECKSUM=5366d7e7fa7ba0d915014d387b66d0d002c03236448e1ba9ef98122c13b35c36
```

Comment:
JFYI, abseil very recently [released](https://github.com/abseil/abseil-cpp/releases/tag/20230802.0) a new LTS version.

Not sure if you're interested in upgrading this quickly, but presumably that'd mean you'd save half a year until the next abseil bump